.github/workflows/branch_pages_deploy.yml

```diff
@@ -37,24 +37,26 @@ jobs:
         with:
           script: |
             const downloadArtefact = require('./.github/workflow-scripts/gh-pages/download-branch-deploy-artefact.js');
-            const download = await downloadArtefact({
-              sha: ${{ fromJSON(steps.prData.outputs.sha }},
+            await downloadArtefact({
```

Comment:
@elenagarrone Spelling here and on line 39

Reply:
Updated to `artifact` 👍